src/sentry/utils/suspect_resolutions/metric_correlation.py

```diff
@@ -71,4 +79,4 @@ def calculate_pearson_correlation_coefficient(x: List[int], y: List[int]) -> flo
     if st_dev_x_y == 0 or st_dev_x_y == 0.0:
         return 0.0
 
-    return cov / st_dev_x_y
+    return float(cov / st_dev_x_y)
```

Comment:
Same as below - isn't this already a float?

Reply:
Same reason for above.